### PR TITLE
Repo cleanup March '21

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #23124. This PR deals with orphaned articles (delete or add to TOC) and orphaned images.

### CODE REVIEW

File: docs/visual-basic/language-reference/error-messages/bc36645.md [hunk 1]
```diff
@@ -1,47 +0,0 @@
----
```
Comment (gewarren) [inline docs/visual-basic/language-reference/error-messages/bc36645.md:1]: @Youssef1313 recently added this file (to replace BC32051) in #19928, however, the error message is already covered by https://github.com/dotnet/docs/blob/main/docs/visual-basic/misc/bc36648-bc36645.md.

**Review by Youssef1313 — COMMENTED** [review]
I'm not sure whether some of the images (for example `upgradereport_plus`) should be deleted or not.

While the image isn't used in articles, it's used by the sample.

https://github.com/dotnet/docs/blob/ac15aa1802911baa07f006ddb2bee06d864b164d/samples/snippets/csharp/VS_Snippets_CFX/operationbehaviorattribute_impersonation/cs/_upgradereport_files/upgradereport.xslt#L185

I haven't checked the other images, but I expect more to be the same.

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

You can :shipit: when ready.

**Comment by gewarren** [comment]
Thanks Bill!

**Pull request merged**